### DISCUSSION TITLE
MissionControl 0.11.0

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#797] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.11.0/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

This is the twenty-third official release of Mission Control.

This release adds support for firmware 18.0.0, a new controller, and binary patches allowing Android devices running Joycon-Droid to operate on firmware 12.0.0+ without the need for root access. See below for full changelog.

**Note: this release requires atmosphere 1.7.0 or later**

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

_PS: there is some ongoing testing of preliminary Bluetooth LE  and USB support happening in my discord server (see #testing-ble and #testing-usb channels). If you own any of the controllers listed [here](https://github.com/ndeadly/MissionControl/issues?q=is%3Aissue+is%3Aopen+label%3A%22BLE+controller%22) or any BLE or USB controllers that are not on the list, we'd like to hear from you! (though I don't need any additional input on Xbox and Stadia controllers for now). Please note that these are alpha builds for the purpose of gathering controller information and furthering my understanding of how BLE/USB works on the console. They are **_NOT_** intended for general use. There _will_ be bugs and issues. There are no ETAs. I will not be providing any support - if you can't follow the usage instructions, then this is not for you._

### Changelog:
- Added full support for firmware 18.0.0.
- Added support for SCUF Vantage 2 controller.
- Added binary patches to enable Joycon-Droid to operate without root access on firmware 12.0.0+.
- Fixed an issue where analog trigger values were swapped for 8BitDo SN30 Pro Xbox Cloud Gaming Edition controller.

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.11.0'>MissionControl 0.11.0</a>.</em>

**Comment by SADEKFCB** [comment]
thank you its work perfect with fw 18.0.0

**Comment by LordFlaya** [comment]
Latest release results in fatal error on 17.0.1 AMS 1.6.2.
Reverting to prior release solves the issue.

**Reply by ndeadly** [reply]
You need to update atmosphere to version 1.7.0

**Comment by luka48** [comment]


**Reply by ndeadly** [reply]
That's not a mission control issue. Something is wrong with your cfw setup

**Comment by Leohw5** [comment]
My Xbox Series controller still can't connect to the Switch....

**Reply by ndeadly** [reply]
Xbox Series controllers use Bluetooth LE, which is not currently supported by the release builds

**Reply by feng2333333333** [reply]
When will it be supported

**Reply by ndeadly** [reply]
In the future. There is already experimental (read: incomplete/unstable) support available in test builds in my discord server

**Reply by jangsili** [reply]
My Xbox controller is in version 3.xx.
It can be connected during Switch Firmware 16.1.0.  But it's not working in firmware 18.0.0.

**Reply by ndeadly** [reply]
Nothing has changed between these firmwares that would affect compatibility. Most likely there is some problem with your updated mission control install

**Reply by jangsili** [reply]
Thanks.

**Reply by jangsili** [reply]
I tried in Firmware 18 and fresh atmosphere 1.7.0 but it is still not work.
Then I downloaded the last version “MissionControl 0.11.0” and put all files into sdcard.
No good.

**Reply by Leohw5** [reply]
Looking forward to support Bluetooth LE.

**Reply by jangsili** [reply]
![image](https://github.com/ndeadly/MissionControl/assets/54981345/b2621977-69df-452a-adb4-4d5d15000120)

**Reply by ndeadly** [reply]
@jangsili this is a legacy firmware version and should work without BLE support. As I said, it's probably a problem with your install. Maybe Mission Control isn't even running. Things you can try:
- Make sure your controller battery level isn't low
- Fix the archive bits on your SD card using hekate
- Delete `/config/MissionControl/controllers` in case it is corrupted

**Comment by Finjii** [comment]
After installing the latest update neither of my PS4 controllers can connect to switch, I tried reinstalling MissionControl as well as updating Atmosphere, Hekate and firmware to 18.0. One controller that was already remembered by the console has its LED turn lavender when connected, but no input is registered :(
Any help would be appreciated

**Reply by ndeadly** [reply]
Sounds like Mission Control might not actually be running. Try fixing the archive bits with hekate

**Reply by jangsili** [reply]
> y fixing the archive bits with hekate

Thanks. It's working now!

**Comment by ReyHarkyon** [comment]
Could you add the 8bitdo NEO GEO control?
